doc/HttpLuaModule.wiki

```diff
@@ -2845,6 +2845,38 @@ The support for the <code>TLSv1.3</code> parameter requires version <code>v0.10.
 
 This directive was first introduced in the <code>v0.9.11</code> release.
 
+== lua_ssl_certificate ==
+
+'''syntax:''' ''lua_ssl_certificate <file>''
+
+'''default:''' ''no''
+
+'''context:''' ''http, server, location''
+
+Specifies the file path to the SSL/TLS certificate in PEM format used for the [[#tcpsock:sslhandshake|tcpsock:sslhandshake]] method.
+
+This directive allows you to specify the SSL/TLS certificate that will be presented to server during the SSL/TLS handshake process.
+
+This directive was first introduced in the <code>v0.10.25</code> release.
```

Comment:
```suggestion
This directive was first introduced in the <code>v0.10.26</code> release.
```

Reply:
ok，i update the version in the doc ，and should i merge all my branches?

Reply:
> 

@zhuizhuhaomeng